.github/workflows/ci_unused_deps.yml

```diff
@@ -0,0 +1,46 @@
+name: Unused Rust crate check
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.head_ref || 'ci_unused_deps' }}
```

Comment:
this needs to be in terms of `dep-check-${{ github.ref }}` or something like that.